tests/mocks.cpp

```diff
@@ -124,6 +125,17 @@ void setup_mock_bpftrace(MockBPFtrace &bpftrace)
   bpftrace.structs.Lookup("struct _tracepoint_file_filename")
       .lock()
       ->AddField("filename", ptr_type, 8, std::nullopt, false);
+  bpftrace.structs.Add("struct _tracepoint_btf_tag", 8);
```

Comment:
maybe make this struct a bit more realistic just in case? right now you've basically made a union, right?
```suggestion
  bpftrace.structs.Add("struct _tracepoint_btf_tag", 16);
```